init-service/log.txt

```diff
@@ -0,0 +1,2 @@
+2021-03-14T11:08:29.057425400Z: Hello, world!
```

Comment:
This file shouldn't be here...

Reply:
oops :) removed

Reply:
I've also added code to clean up the file as part of the test